### PR TITLE
e2e-gcp-ipi-sdn-migration-ovn-p2-update

### PR DESCRIPTION
**Pull request by weliang1**
#32778 is messed up. Open a new one to do the same task.

@anuragthehatter @zhaozhanqi @lihongan PTAL!

### CODE REVIEW

**Comment by weliang1** [comment]
Do not know why `ci/prow/generated-config ` keep failing, even run `make jobs`several times.

**Comment by weliang1** [comment]
> Do not know why `ci/prow/generated-config ` keep failing, even run `make jobs`several times.

@jianlinliu Could you help to review this PR and find out any reason why `ci/prow/generated-config ` keep failing? Thanks!

**Comment by jianlinliu** [comment]
It is explained in https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/33140/pull-ci-openshift-release-master-generated-config/1581108484756213760
```
ERROR: This check enforces that Prow Job configuration YAML files are generated
ERROR: correctly. We have automation in place that generates these configs and
ERROR: new changes to these job configurations should occur from a re-generation.
ERROR: Run the following command to re-generate the Prow jobs:
ERROR: $ make jobs
```

**Comment by zhaozhanqi** [comment]
> Do not know why `ci/prow/generated-config ` keep failing, even run `make jobs`several times.

then it's weird.  Could you double confirm if you push after `make jobs` or any errors happened when `make jobs`

**Comment by weliang1** [comment]
The failure may due to some upgrade pre and post check case, not work flow itself.
INFO[2022-10-18T21:43:33Z] Gathering overall upgrade sanity test results
2 failures in upgrade prepare
4 failures in upgrade check
Please investigate these failures from build artifacts
{"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"k8s.io/test-infra/prow/entrypoint/run.go:79","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2022-10-18T21:43:23Z"}
error: failed to execute wrapped command: exit status 1 
INFO[2022-10-18T21:43:33Z] Step e2e-gcp-ipi-sdn-migration-ovn-p2-cucushift-upgrade-report failed after 1m10s. 
INFO[2022-10-18T21:43:33Z] Step phase test failed after 1h33m20s.       
INFO[2022-10-18T21:43:33Z] Running multi-stage phase post

**Comment by zhaozhanqi** [comment]
> The failure may due to some upgrade pre and post check case, not work flow itself. INFO[2022-10-18T21:43:33Z] Gathering overall upgrade sanity test results 2 failures in upgrade prepare 4 failures in upgrade check Please investigate these failures from build artifacts {"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"k8s.io/test-infra/prow/entrypoint/run.go:79","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2022-10-18T21:43:23Z"} error: failed to execute wrapped command: exit status 1 INFO[2022-10-18T21:43:33Z] Step e2e-gcp-ipi-sdn-migration-ovn-p2-cucushift-upgrade-report failed after 1m10s. INFO[2022-10-18T21:43:33Z] Step phase test failed after 1h33m20s. INFO[2022-10-18T21:43:33Z] Running multi-stage phase post

@weliang1 since the cluster is not enable fips.  I guess we can add `not @fips` for the tag

**Comment by weliang1** [comment]
> > The failure may due to some upgrade pre and post check case, not work flow itself. INFO[2022-10-18T21:43:33Z] Gathering overall upgrade sanity test results 2 failures in upgrade prepare 4 failures in upgrade check Please investigate these failures from build artifacts {"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"k8s.io/test-infra/prow/entrypoint/run.go:79","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2022-10-18T21:43:23Z"} error: failed to execute wrapped command: exit status 1 INFO[2022-10-18T21:43:33Z] Step e2e-gcp-ipi-sdn-migration-ovn-p2-cucushift-upgrade-report failed after 1m10s. INFO[2022-10-18T21:43:33Z] Step phase test failed after 1h33m20s. INFO[2022-10-18T21:43:33Z] Running multi-stage phase post
> 
> @weliang1 since the cluster is not enable fips. I guess we can add `not @fips` for the tag for the failed cases

@zhaozhanqi With `not @fips` tag, some test cases still failed. It may be the bugs for the features migration. I will do more manual testing for more investigation.  Seems the whole testing processes are working fine with this work flow, we can merge this PR and continue working on the rests. WDYT?

**Comment by zhaozhanqi** [comment]
> > > The failure may due to some upgrade pre and post check case, not work flow itself. INFO[2022-10-18T21:43:33Z] Gathering overall upgrade sanity test results 2 failures in upgrade prepare 4 failures in upgrade check Please investigate these failures from build artifacts {"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"k8s.io/test-infra/prow/entrypoint/run.go:79","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2022-10-18T21:43:23Z"} error: failed to execute wrapped command: exit status 1 INFO[2022-10-18T21:43:33Z] Step e2e-gcp-ipi-sdn-migration-ovn-p2-cucushift-upgrade-report failed after 1m10s. INFO[2022-10-18T21:43:33Z] Step phase test failed after 1h33m20s. INFO[2022-10-18T21:43:33Z] Running multi-stage phase post
> > 
> > 
> > @weliang1 since the cluster is not enable fips. I guess we can add `not @fips` for the tag for the failed cases
> 
> @zhaozhanqi With `not @fips` tag, some test cases still failed. It may be the bugs for the features migration. I will do more manual testing for more investigation. Seems the whole testing processes are working fine with this work flow, we can merge this PR and continue working on the rests. WDYT?

ok for me to merge this, it's not the migration issue, please continue investigating this. 
/lgtm

**Comment by weliang1** [comment]
> > > > The failure may due to some upgrade pre and post check case, not work flow itself. INFO[2022-10-18T21:43:33Z] Gathering overall upgrade sanity test results 2 failures in upgrade prepare 4 failures in upgrade check Please investigate these failures from build artifacts {"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"k8s.io/test-infra/prow/entrypoint/run.go:79","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2022-10-18T21:43:23Z"} error: failed to execute wrapped command: exit status 1 INFO[2022-10-18T21:43:33Z] Step e2e-gcp-ipi-sdn-migration-ovn-p2-cucushift-upgrade-report failed after 1m10s. INFO[2022-10-18T21:43:33Z] Step phase test failed after 1h33m20s. INFO[2022-10-18T21:43:33Z] Running multi-stage phase post
> > > 
> > > 
> > > @weliang1 since the cluster is not enable fips. I guess we can add `not @fips` for the tag for the failed cases
> > 
> > 
> > @zhaozhanqi With `not @fips` tag, some test cases still failed. It may be the bugs for the features migration. I will do more manual testing for more investigation. Seems the whole testing processes are working fine with this work flow, we can merge this PR and continue working on the rests. WDYT?
> 
> ok for me to merge this, it's not the migration issue, please continue investigating this. /lgtm

@zhaozhanqi 
Three issues were opened for one feature and two upgrade cases:
https://issues.redhat.com/browse/OCPQE-12577
https://issues.redhat.com/browse/OCPQE-12578
https://issues.redhat.com/browse/OCPBUGS-2665

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -1007,6 +1007,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-upi-disconnected
+- as: e2e-gcp-ipi-sdn-migration-ovn-p2
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:1010]: could you sort the jobs ?

**Comment by weliang1** [comment]
Seems a new https://github.com/openshift/release/pull/33374 is doing the same job as this one

**Pull request closed without merging**